frontend/public/components/RBAC/bindings.jsx

```diff
@@ -298,7 +299,7 @@ const NsDropdown_ = props => {
   const resources = [{ kind }];
   return <ListDropdown {...props} desc="Namespaces" resources={resources} selectedKeyKind={kind} placeholder="Select namespace" />;
 };
-const NsDropdown = connectToFlags(FLAGS.OPENSHIFT)(NsDropdown_);
+export const NsDropdown = connectToFlags(FLAGS.OPENSHIFT)(NsDropdown_);
```

Comment:
hmm this is a bit tricky... `NamespaceDropdown` is used in the `NamespaceSelector`. Semantically speaking we could place the `NsDropdown` together with underlying component into the *namespace.jsx*, together with proper comments, to avoid confusion (generally speaking it would come handy to have comments on a lot of out components, eg: `Firehose`, ...). Advantage would be that both are already in `.jsx` format.
If we decide not to put it there I would suggest to create a *namespace-dropdown*. Then the question is if we want to revrite it into `.tsx` or just put it into `.jsx`.
Also we want do do it as part of this PR :question: